scripts/quests/crystalWar/On_Sabbatical.lua

```diff
@@ -0,0 +1,134 @@
+-----------------------------------
+-- On Sabbatical
+-----------------------------------
+-- !addquest 7 32
+-- Erlene                : !pos 376.936 -39.999 17.914 175
+-- Gentle Tiger          : !pos -203.932 -9.998 2.237 87
+-- Indescript Markings   : !pos -456.707 24.4385 -363.364 90
+-----------------------------------
+require('scripts/globals/npc_util')
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.CRYSTAL_WAR, xi.quest.id.crystalWar.ON_SABBATICAL)
+
+quest.reward =
+{
+    item = xi.items.KLIMAFORM_SCHEMA
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainJob() == xi.job.SCH and
+                player:getMainLvl() >= xi.settings.main.AF1_QUEST_LEVEL
+        end,
+
+        [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
+        {
+            ['Erlene'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(18)
+                end
+            },
+
+            onEventFinish =
+            {
+                [18] = function(player, csid, option, npc)
+                    local ID = zones[player:getZoneID()]
+                    if npcUtil.giveKeyItem(player, xi.ki.ULBRECHTS_SEALED_LETTER) then
```

Comment:
Probably don't need the condition here, but it doesn't hurt

Reply:
done